elif.py

```diff
@@ -0,0 +1,16 @@
+## using elif statement instead of multiple if statements
```

Comment:
GPT summary of [None](https://github.com/SampatSuthar/ml-demo/blob/2ce9a211365f18cfb46299b15c0e532c3e8af178/elif.py#None) - [74c990](https://github.com/SampatSuthar/ml-demo/blob/8c769ca7623fda20a6e3ff1242f88e2398d4b453/elif.py#74c99050e04e78a20eced13dbd686eb4c597e0e4):
* Added code to use an elif statement instead of multiple if statements
* Added code to test with GPT summary
* Added code to set petrol prices for Rajasthan, Maharashtra, and Gujarat
* Added code to set the petrol price for other states